test/filters/asCompanyTypeGoalId.filter.test.ts

```diff
@@ -0,0 +1,24 @@
+import { assert } from 'chai'
+
+import { asCompanyTypeGoalId } from 'app/filters/asCompanyTypeGoalId.filter'
+import ClosableCompanyType from 'app/models/mapper/closableCompanyType.enum'
+import PiwikConfig from 'app/models/piwikConfig'
+
+const piwikConfig: PiwikConfig = {
+  url: '',
+  siteId: '',
+  landingPageStartGoalId: 999,
+  confirmationPagePDFGoalId: 999,
+  limitedCompanyGoalId: '',
```

Comment:
This field should have a value, instead of confirmationPagePDFGoalId. Also, the value should be different for the both of the fields